packages/@react-native-windows/cli/src/healthChecks.ts

```diff
@@ -19,7 +36,7 @@ export function getHealthChecks(): HealthCheckCategory[] | undefined {
         'react-native-windows/package.json',
         {paths: [process.cwd()]})), 'Scripts/rnw-dependencies.ps1');
     
-    const rnwDeps = execSync(`powershell -ExecutionPolicy Unrestricted -NoProfile ${rnwDepScriptPath} -NoPrompt -ListChecks`);
+    const rnwDeps = execSync(`powershell -ExecutionPolicy Unrestricted -NoProfile ${rnwDepScriptPath} -NoPrompt -ListChecks`, {stdio: 'pipe'});
```

Comment:
Does the pipe here end up spewing a bunch of output into the normal `react-native doctor` command?  I understand you are probably trying to gain some useful output for cases where the script fails.  Just want to understand if we are making a trade off of polluting the normal output?

Reply:
Opposite. By default execSync will output `stderr` to the parent process output. `inherit` directs it just to the subprocess stdout, stderr, etc.

Reply:
We previously got this gnarly output. 
![image](https://user-images.githubusercontent.com/835219/135146259-92be0ffa-af79-4223-bac4-20fcb925867e.png)

Reply:
After change: 
![image](https://user-images.githubusercontent.com/835219/135146316-e8c42dc4-557d-4663-b51c-9ab283a456d1.png)

Reply:
Verified the correct case still works.
![image](https://user-images.githubusercontent.com/835219/135146380-8bf4eb85-4f6c-4889-abab-d1a4db257ef7.png)

Reply:
This falls further into the "eating the error" territory, but folks can always run the deps script manually to see the full output of why it is failing.